src/factorio_server.go

```diff
@@ -43,6 +43,27 @@ func randomPort() int {
 	return rand.Intn(45000-40000) + 40000
 }
 
+func autostart() {
+
+	var err error
+	if FactorioServ.BindIP == "" {
+		FactorioServ.BindIP = "0.0.0.0"
+
+	}
+	if FactorioServ.Port == 0 {
+		FactorioServ.Port = 34197
+	}
+	FactorioServ.Savefile = "Load Latest"
+
+	err = FactorioServ.Run()
+
+	if err != nil {
+		log.Printf("Error starting Factorio server: %+v", err)
```

Comment:
I guess the `%+v` is a typo.
```suggestion
		log.Printf("Error starting Factorio server: %v", err)
```

Reply:
from the golang docs:
```
%v	the value in a default format
	when printing structs, the plus flag (%+v) adds field names
```

Reply:
We use `%v`, `%+v` and `%s` already in the code, so i dont see a problem there. Also this is only something that is printed in the logs, so i dont see a problem there.